### ISSUE TITLE
kstest not working

### ISSUE THREAD

**Issue by cperk** [spatstat/spatstat#87]
I would like to run a Kolmogorov-Smirnov test (kstest) on my ppp object as described in:

[https://training.fws.gov/courses/references/tutorials/geospatial/CSP7304/documents/PointPatterTutorial.pdf](url)

However, I keep getting this error:

`Error in kstest(cells, KST.cells) : could not find function "kstest"`

Here is a reproducible example. 

Examine ppp object called "cells" and plot density:

`data(cells)`
`par(mfrow = c(1, 2))`
`plot(cells)`
`plot(density(cells))`

![toy latest](https://user-images.githubusercontent.com/21315115/48218834-cb2ab100-e358-11e8-95bb-3bda46dbf00d.png)

Attempt to run kstest command and receive error message:

`KST.cells <- function(x,y) { x }`
`kstest(cells, KST.cells)`

`Error in kstest(cells, KST.cells) : could not find function "kstest"`

Is there anything I am doing wrong? How can I get this working?

**Comment by rubak** [comment]
You do realize that these notes are from a decade ago when R was version 2.6 (now 3.5) and spatstat version 1.12 (now 1.57) which is very long ago for software?

Anyway, most of the material probably works fine, but be aware: there may be many things that can be done in smarter ways, there are tons of additions that are not described there, and there may be a few functions that have been renamed or taken over by more general functions (this is what happened to `kstest` which was superseded by `cdf.test` in version 1.38 in 2014, but only recently completely removed in early 2018 after years of warnings).

The recommended way to learn how to use spatstat these days is from [the book](http://book.spatstat.org/), but if you don't have access to it you may also learn from [a workshop held in 2017](http://spatstat.org/SSAI2017/).

Does this solve your problem? Then I will go ahead and close the issue...

**Comment by cperk** [comment]
Yes, thank you. I realized that it was an old document, but had already
used many of the other tutorials online and thought I would check it out. I
will definitely look into the book and workshop.

So just to clarify, there is no way to do that particular test anymore?

**Comment by rubak** [comment]
Yes, you can still do the Kolmogorov-Smirnov (K-S) test, but it is only one of several options in the new `cdf.test` function. It basically works the same way as before and the K-S test is the default, so you just need to replace `kstest` by `cdf.test` in your final line.
